r/tests/testthat/test-dplyr.R

```diff
@@ -421,12 +484,13 @@ test_that("explicit type conversions", {
         int2dbl = as.double(int),
         int2int = as.integer(int),
         int2lgl = as.logical(int),
-        lgl2chr = toupper(as.character(lgl)), # Arrow returns "true", "false"
+        lgl2chr = as.character(lgl), # Arrow returns "true", "false" here ...
         lgl2dbl = as.double(lgl),
         lgl2int = as.integer(lgl),
-        lgl2lgl = as.logical(lgl),
+        lgl2lgl = as.logical(lgl)
       ) %>%
-      collect(),
+      collect() %>%
+      mutate(lgl2chr = toupper(lgl2chr)), # ... but we need "TRUE", "FALSE"
```

Comment:
I moved the `toupper()` to after `collect()` to allow this test to run successfully without utf8proc. This fixes a `test-r-minimal-build` failure

Reply:
Can you add a comment explaining that?

Reply:
Added in 95272bc268a5742ecf3a79828172f0de75801bc7